public/app/features/explore/NoDataSourceCallToAction.tsx

```diff
@@ -29,8 +29,8 @@ export const NoDataSourceCallToAction = () => {
   );
 
   const cardClassName = css`
-    max-width: ${theme.breakpoints.lg};
-    margin-top: ${theme.spacing.md};
+    max-width: ${theme.breakpoints.values.lg};
```

Comment:
Based on https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/themes/breakpoints.ts#L6 the value is number, therefore we need `px` behind it. Here is the example https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/DateTimePickers/DateTimePicker/DateTimePicker.tsx#L27

Reply:
Ah i see, thanks for the heads up!